### PR TITLE
Feature/scenario3.5

### PR DESCRIPTION
**Pull request by anton-sidelnikov**
- [x] infrastructure: 2 ecs's with eip, one for target(host) iscsi, another for initiator(user)

Hosts setup will be in another branch

### CODE REVIEW

File: scenarios/scenario3_5/resources/first_boot.sh [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#!/usr/bin/env bash
```
Comment (outcatcher) [inline scenarios/scenario3_5/resources/first_boot.sh]: Is this file required?

File: scenarios/scenario3_5/resources/variables.tf [hunk 2]
```diff
@@ -0,0 +1,16 @@
+variable "volume_type" { default = "SATA" }
+variable "key_pair" {
+  type = object({
+    key_name : string
+    public_key : string
+  })
+}
+variable "debian_image" {}
```
Comment (outcatcher) [inline scenarios/scenario3_5/resources/variables.tf]: Please rename to `ecs_image`

File: scenarios/scenario3_5/resources/variables.tf [hunk 3]
```diff
@@ -0,0 +1,16 @@
...
+variable "nodes_count" {}
+variable "default_flavor" {}
+variable "disc_volume" {}
+variable "subnet" {}
+variable "network" {}
+variable "prefix" {}
+variable "default_az" {}
```
Comment (outcatcher) [inline scenarios/scenario3_5/resources/variables.tf]: Please rename to `availability_zone` to make it more understandible

File: scenarios/scenario3_5/variables.tf [hunk 4]
```diff
@@ -0,0 +1,19 @@
+variable "username" {}
+variable "password" {}
+variable "region" {}
+variable "tenant_name" {}
+variable "default_az" {}
+variable "domain_name" {}
+variable "default_flavor" {}
```
Comment (outcatcher) [inline scenarios/scenario3_5/variables.tf]: Please rename to `ecs_flavor` to make it more understandible

File: scenarios/scenario3_5/variables.tf [hunk 5]
```diff
@@ -0,0 +1,19 @@
+variable "username" {}
+variable "password" {}
+variable "region" {}
+variable "tenant_name" {}
+variable "default_az" {}
```
Comment (outcatcher) [inline scenarios/scenario3_5/variables.tf]: Please rename to `availability_zone` to make it more understandable

File: scenarios/scenario3_5/variables.tf [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+variable "domain_name" {}
+variable "default_flavor" {}
+variable "debian_image" {}
```
Comment (outcatcher) [inline scenarios/scenario3_5/variables.tf]: Please rename to `ecs_image` to make it more understandable

File: scenarios/scenario3_5/resources/first_boot.sh (hunk 1 above)
Comment (anton-sidelnikov) [reply scenarios/scenario3_5/resources/first_boot.sh]: no
